src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4495,14 +4495,37 @@ static InitialSessionState()
             builtinVariables.Add(
                 new SessionStateVariableEntry(
                     SpecialVariables.NativeArgumentPassing,
-                    Platform.IsWindows ? NativeArgumentPassingStyle.Windows : NativeArgumentPassingStyle.Standard,
+                    GetPassingStyle(), // Platform.IsWindows ? NativeArgumentPassingStyle.Windows : NativeArgumentPassingStyle.Standard,
```

Comment:
Shall we remove the comment?
```suggestion
                    GetPassingStyle(),
```

Reply:
@SteveL-MSFT 
Yes, that is correct. We should be defaulting to legacy behavior in stable releases